src/Lykke.Service.ExchangeConnector/Repositories/OrderBookEventEntity.cs

```diff
@@ -31,8 +31,9 @@ public OrderBookEventEntity(string orderBookSnapshotId, DateTime orderEventTimes
             OrderEventTimestamp = orderEventTimestamp;
             OrderId = orderId;
 
+            Timestamp = DateTime.UtcNow;
```

Comment:
Azure Storage Table Entity's Timestamp field is maintained by the server, it will override it anyway for last modified timestamp, and AFAIK the Timestamp field is not an indexed one. Probably it's better to keep it as it was, including timestamp into the RowKey.

Reply:
Yes, Timestamp was filled only for debugging purposes. It will be rewritten by the server.
About the rowkey - we might have two orders with the same orderid and timestamp. Even if we include the eventid, there still can be duplicates - that's why I've changed it to Guid. Having the timestamp included in the RowKey will increase the data size, and I don't think if we are going to have exact searches by RowKey/Timestamp.